### DISCUSSION TITLE
Change transition easing to an ease

### DISCUSSION THREAD

**Discussion by probablykasper** [saadeghi/daisyui#1938] [Ideas / feature requests]
Most of the DaisyUI transitions feel slow/unresponsive to me. I think it would be a great improvement to switch to an easing function that eases out only.

I really like the the ease-out that Tailwind uses, `cubic-bezier(0, 0, 0.2, 1)`. Here's a comparison:

https://github.com/saadeghi/daisyui/assets/11315492/9341e368-b909-4c6c-8649-851e2cbf8b23

**Comment by saadeghi** [comment]
Transition and animation improvements applied to components in daisyUI `3.5.0`  
Thanks for the suggestion
